wiki/Beatmap_Editor/Song_Setup/fr.md

```diff
@@ -1,193 +1,259 @@
+
+*Voir également : [\[Guide\] Metadata](https://osu.ppy.sh/community/forums/topics/249288 "osu! forums")*
+
+![A completed Song Setup form](img/SS_General.jpg "Un exemple de métadonnées complètes (onglet Général) dans la configuration de la chanson")
+
+Ces champs doivent être remplis avant de créer une deuxième difficulté. Ceci est principalement pour s'assurer que les métadonnées seront cohérentes à travers toutes les difficultés du beatmapset.
+
+#### Artiste
+
+**Le groupe, le chanteur, le compositeur ou le groupe qui a interprété ou créé la chanson.** Normalement, cela est automatiquement pris dans les métadonnées du fichier MP3, mais vérifiez l'orthographe et la ponctuation.
+
+*Avis : Les titres d'anime ou de jeu appartiennent à la catégorie [Source](#source). Les noms des vocaloïdes ou des sociétés appartiennent à la catégorie [Tags](#tags)*
+
+Les noms étrangers/orientaux doivent être présentés dans le même ordre que celui utilisé sur Wikipédia (généralement, le prénom suivi du nom de famille ; l'ordre occidental, par souci de cohérence).
+
+Ce champ doit être préféré à l'orthographe des sources de diffusion officielles. L'artiste peut parfois être trouvé sur [Wikipedia](https://www.wikipedia.org) ou d'autres sources non officielles telles que [Anime News Network](https://www.animenewsnetwork.com) ou [MyAnimeList](https://myanimelist.net). Comme pour toute source non officielle, le contenu fourni doit être vérifié par recoupement avec les sources officielles ou, s'il fournit un lien vers la source officielle, il faut plutôt consulter la source officielle. En cas de doute, remplissez toutes les informations connues maintenant et faites un post de demande de vérification.
+
+##### Artiste romanisé
+
+**Le nom romanisé de l'artiste** doit être présenté dans le même ordre que celui utilisé sur Wikipédia (généralement, le prénom suivi du nom de famille ; l'ordre occidental, par souci de cohérence). Ce champ ne sera disponible que si des caractères Unicode sont utilisés dans la catégorie [Artist](#artist).
+
+#### Titre
+
+**Le titre de la chanson**. Normalement, il est automatiquement extrait des métadonnées du fichier MP3, mais vérifiez l'orthographe et la ponctuation.
+
+##### Titre romanisé
+
+**Le nom romanisé de la chanson.** Ce champ ne sera disponible que si des caractères Unicode sont utilisés dans la catégorie [Title](#title).
+
+#### Créateur de Beatmap
 
-# Paramètres de beatmap
+**Le nom d'utilisateur du créateur de la beatmap.** Le client du jeu devrait gérer cela automatiquement et ne vous permettra pas de modifier ce champ, c'est-à-dire si vous êtes le créateur.
 
-Les paramètres d'une beatmap est une étape importante dans le [beatmapping](/wiki/Beatmapping), puisqu'il contient des paramètres importants qui sont les clés pour créer une map correcte.
+*Notice: Les noms d'utilisateur de [Guest Creators](/wiki/GD) appartiennent à la catégorie [Difficulty](#difficulty) et/ou [Tags](#tags).*
```

Comment:
Also, because you translated the headings, the heading ids also change to what you translated them to.

What this means is that `#difficulty` and `#tags` will go nowhere when viewing this translation.

```suggestion
*Notice: Les noms d'utilisateur de [Guest Creators](/wiki/GD) appartiennent à la catégorie [Difficulty](#difficulté) et/ou [Tags](#balises).*
```